packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -256,10 +256,17 @@ Image.propTypes = {
   }).isRequired,
 };
 
+// This styled component is used to make sure the tooltip is above the popover
 const TooltipCustom = styled(Tooltip)`
   z-index: 6;
 `;
 
+const ButtonLink = styled(Button)`
+  & > span {
+    line-height: normal;
+  }
+`;
```

Comment:
I find the name misleading since this is not a link. I don't have great suggestions though, maybe CustomButton or ButtonWrapper?

Reply:
`NormalizedLineHeightButton`? Does what it says on the tin lol